Assets/Scripts/MultiplayerGames.cs

```diff
@@ -167,7 +167,7 @@ public class MultiplayerGames : MonoBehaviour {
 				Game.SwitchScene ("RRReadyScene");
 			} else if (args.gameID == Constants.MINIGAME_CARDS_OF_WILD) {
 				CW.GameManager.matchID = args.id;
-				NetworkManager.Send (CW.MatchInitProtocol.Prepare 
+				Game.networkManager.Send (CW.MatchInitProtocol.Prepare 
```

Comment:
This should look like this:

CW.GameManager.matchID = args.id;
gameObject.AddComponent <CWGame>();
CWGame.networkManager.Send (CW.MatchInitProtocol.Prepare 
				                        (GameState.player.GetID(), args.id), 
				                        ProcessMatchInit);

And @rupalkhilari is right. The port number needs to be 20038 in CWAssets->Scripts->Constants.cs